feathr_project/test/test_user_workspace/feathr_config.yaml

```diff
@@ -102,6 +102,7 @@ online_store:
     ssl_enabled: True
 
 feature_registry:
+  endpoint: "https://feathr-sql-registry.azurewebsites.net/api/v1"
   purview:
```

Comment:
nit: `rest_endpoint` to make it a bit more reflective?

Also I'm not sure if the purview section is needed or not if we are using REST API, given that those purview account info will be maintained in the REST API layer?

Reply:
Renamed to `api_endpoint`